ext/random/engine_combinedlcg.c

```diff
@@ -32,17 +32,17 @@
  */
 #define MODMULT(a, b, c, m, s) q = s / a; s = b * (s - a * q) - c * q; if (s < 0) s += m
 
-static void seed(php_random_status *status, uint64_t seed)
+static void seed(void *status, uint64_t seed)
```

Comment:
```suggestion
static void seed(php_random_status_state_combinedlcg *status, uint64_t seed)
```

why is void pointer better than a "typed pointer"?

Reply:
Because the code compiles.